test/parallel/test-trace-events-async-hooks-dynamic.js

```diff
@@ -0,0 +1,83 @@
+'use strict';
+
+const common = require('../common');
+if (!process.binding('config').hasTracing)
+  common.skip('missing trace events');
+
+const assert = require('assert');
+const cp = require('child_process');
+const fs = require('fs');
+const path = require('path');
+const util = require('util');
+
+const enable = `require("trace_events").createTracing(
+{ categories: ["node.async_hooks"] }).enable();`;
+const code =
+  'setTimeout(() => { for (var i = 0; i < 100000; i++) { "test" + i } }, 1)';
+
+const tmpdir = require('../common/tmpdir');
+const filename = path.join(tmpdir.path, 'node_trace.1.log');
+
+tmpdir.refresh();
+const proc = cp.spawnSync(
+  process.execPath,
+  ['-e', enable + code ],
+  {
+    cwd: tmpdir.path,
+    env: {
+      'NODE_DEBUG_NATIVE': 'tracing',
+      'NODE_DEBUG': 'tracing'
+    }
```

Comment:
This will fail on some platforms (e.g. Linuxone) in the CI where `LD_LIBRARY_PATH` is used to point to the C++ runtime libraries. The tests should extend `env` rather than replace it.